ViacheslavBlazhkov/HomeWork_4/Program.cs

```diff
@@ -0,0 +1,93 @@
+﻿class HomeWork4
+{
+    // Task 1 ------------------------------------------
+    static int MaxValue(params int[] numbers)
+    {
+        int max = numbers[0];
+        for (int i = 1; i < numbers.Length; i++)
+        {
+            if (numbers[i] > max) max = numbers[i];
+        }
+        return max;
+    }
+
+    // Task 2 ------------------------------------------
+    static int MinValue(params int[] numbers)
+    {
+        int min = numbers[0];
+        for (int i = 1; i < numbers.Length; i++)
+        {
+            if (numbers[i] < min) min = numbers[i];
+        }
+        return min;
+    }
+
+    // Task 3 ------------------------------------------
+    static bool TrySumIfOdd(int first, int second, out int sum)
+    {
+        sum = first + second;
+        if (sum % 2 != 0) return true;
+        else return false;
+    }
+
+// Task 4 ------------------------------------------
+    static int MaxValue(int first, int second, int third)
+    {
+        int max = first;
+        if (first > second && first > third) max = first;
+        else if (second > third) max = second;
+        else max = third;
+        return max;
+    }
+    static int MaxValue(int first, int second, int third, int fourth)
+    {
+        int max = first;
+        if (first > second && first > third && first > fourth) max = first;
+        else if (second > third && second > fourth) max = second;
+        else if (third > fourth) max = third;
+        else max = fourth;
+        return max;
+    }
+    static int MinValue(int first, int second, int third)
+    {
+        int min = first;
+        if (first < second && first < third) min = first;
+        else if (second < third) min = second;
+        else min = third;
+        return min;
+    }
+
+    static int MinValue(int first, int second, int third, int fourth)
```

Comment:
Looks little bit complicated, you can recall ex. MinValue method with 3 param (return first < MinValue(second, third, fourth) ... )